javascripts/content_scripts/privly.js

```diff
@@ -299,8 +298,8 @@ var privly = {
       var notUpdated = [];
       elements.forEach(
         function(a){
-          if ( ! a.textContent.indexOf("privlyInject1") > 0 // Optimization
-            || ! privly.correctIndirection.testAndCopyOver(a, a.textContent) ) {
+          // Optimization
+          if (  a.textContent.indexOf("privlyInject1") < 0 || ! privly.correctIndirection.testAndCopyOver(a, a.textContent) ) {
```

Comment:
I thought that might be the case. It should also complain if the line is too long, but that would likely have too many false positives.